### ISSUE TITLE
Prefer long unicode class names over the short ones

### ISSUE THREAD

**Issue by fisker** [ota-meshi/eslint-plugin-regexp#720]
**Motivation**

The long hand names are more readable.

**Description**

Same as above.

**Examples**

```js
/* ✓ GOOD */
/\p{Letter}/v

/* ✗ BAD */
/\p{L}/v
```

```js
/* ✓ GOOD */
 /\p{Script=Greek}/v

/* ✗ BAD */
/\p{sc=Greek}/v
```

**Comment by ota-meshi** [comment]
Thank you for the rule suggestion! That rule sounds good to me!
